api/v1alpha1/runner_types.go

```diff
@@ -207,6 +210,12 @@ type RunnerStatusRegistration struct {
 	ExpiresAt    metav1.Time `json:"expiresAt"`
 }
 
+type WorkVolumeClaimTemplate struct {
```

Comment:
If `WorkVolumeClaimTemplate` field exists, all of these fields are required. Is it okay?